maintainers/scripts/update.nix

```diff
@@ -105,7 +105,7 @@ let
     to run all update scripts for all packages that lists \`garbas\` as a maintainer
     and have \`updateScript\` defined, or:
 
-        % nix-shell maintainers/scripts/update.nix --argstr package garbas
+        % nix-shell maintainers/scripts/update.nix --argstr package gnome3
```

Comment:
I actually thing several things aren't correct here. This should be like `maintainer $maintainer_attr`, and to run for a specific package below should be `package`. This is out of scope and has various other inaccuracy, can you drop the commit?

Reply:
no, it is real typo. Your suggested variant is already listed https://github.com/NixOS/nixpkgs/blob/master/maintainers/scripts/update.nix#L103-L111

Reply:
but I found that my suggestion was bad as well. I've changed the suggestion and verified it runs